src/aggregator/src/test/resources/au/org/emii/aggregator/cars_weekly_expected.cdl

```diff
@@ -0,0 +1,776 @@
+      :_FillValue = 1.17549435E-38f; // float
+      :long_name = "intra_and_inter-seasonal_mean_of_sea_water_temperature";
+      :units = "Celsius";
+
+    float TEMP_std_dev(DEPTH=2, LATITUDE=1, LONGITUDE=1);
+      :_FillValue = 1.17549435E-38f; // float
+      :long_name = "intra_and_inter-seasonal_weighted_standard_deviation_of_sea_water_temperature";
+      :units = "Celsius";
+      :coordinates = "LATITUDE LONGITUDE DEPTH";
+
+    short TEMP_sumofwgts(DEPTH=2, LATITUDE=1, LONGITUDE=1);
+      :coordinates = "LATITUDE LONGITUDE DEPTH";
+      :_FillValue = -32768S; // short
+      :long_name = "sum_of_weights_for_sea_water_temperature";
+      :units = 1.0; // double
+
+  // global attributes:
+  :abstract = "CARS (CSIRO Atlas of Regional Seas) is a digital climatology, or atlas of seasonal ocean water properties. It comprises gridded fields of mean ocean properties over the period of modern ocean measurement, and average seasonal cycles for that period. It is derived from a quality-controlled archive of all available historical subsurface ocean property measurements - primarily research vessel instrument profiles and autonomous profiling buoys. As data availability has enormously increased in recent years, the CARS mean values are inevitably biased towards the recent ocean state. See http://www.marine.csiro.au/~dunn/cars2009/ .";
+  :acknowledgement = "The User agrees that whenever the Product or imagery/data derived from the Product are published by the User, the CSIRO Marine Laboratories shall be acknowledged as the source of the Product.";
+  :author = "Galibert, Guillaume";
+  :author_email = "guillaume.galibert@utas.edu.au";
+  :citation = "The citation in a list of references is: \"CSIRO [year-of-data-download], [Title], [data-access-url], accessed [date-of-access]\"";
+  :Conventions = "CF-1.6,IMOS-1.4";
+  :data_centre = "Australian Ocean Data Network (AODN)";
+  :data_centre_email = "info@aodn.org.au";
+  :date_created = "2018-01-16T23:35:02Z";
+  :disclaimer = "The User acknowledges that the Product was developed by CSIRO for its own research purposes. The CSIRO will not therefore be liable for interpretation of or inconsistencies, discrepancies, errors or omissions in any or all of the Product as supplied. Any use of or reliance by the User on the Product or any part thereof is at the User\'s own risk and CSIRO shall not be liable for any loss or damage howsoever arising as a result of such use. The User agrees to indemnify and hold harmless CSIRO in respect of any loss or damage (including any rights arising from negligence or infringement of third party intellectual property rights) suffered by CSIRO as a result of User\'s use of or reliance on the Data.";
+  :geospatial_lat_max = -4.0; // double
+  :geospatial_lat_min = -50.0; // double
+  :geospatial_lon_max = 161.0; // double
+  :geospatial_lon_min = 104.0; // double
+  :geospatial_vertical_max = "100";
+  :geospatial_vertical_min = "50";
```

Comment:
Global attributes `geospatial_vertical_min/max` should ideally be of the same type as the DEPTH variable (in this case short, not string). A numeric type like double or float could also be acceptable though.